packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -182,15 +170,15 @@ export const ConnectedServices = () => {
       onCompleted: () => {
         // TODO: Add `timing.clients.disconnect` flow timing event as seen in
         // old-settings? #6903
-        if (DS.client!.isCurrentSession) {
+        if (client.current?.isCurrentSession) {
           clearSignedInAccountUid();
           window.location.assign(`${window.location.origin}/signin`);
-        } else if (DS.reason === 'suspicious' || DS.reason === 'lost') {
+        } else if (reason === 'suspicious' || reason === 'lost') {
```

Comment:
Optional: should we use an enum for the reason, instead of a string?

Reply:
meh